articles/virtual-machines/capture-image-portal.md

```diff
@@ -26,36 +26,38 @@ For images stored in an Azure Compute Gallery (formerly known as Shared Image Ga
 1. Go to the [Azure portal](https://portal.azure.com), then search for and select **Virtual machines**.
 
 2. Select your VM from the list.
+- If you want a generalized image, see [Generalize OS disk for Linux/Windows](https://learn.microsoft.com/azure/virtual-machines/generalize).
 
+   If you want a specialized image, no additional action required,Kindly proceed to Step 3.

```

Comment:
```suggestion
- If you want a specialized image, no additional action is required.
```

Reply:
In addition to Julia's review feedback, please fix the reference link with an absolute path.